build.sbt

```diff
@@ -75,7 +75,26 @@ lazy val server = (projectMatrix in file("server"))
       artifactPath / (Compile / packageBin) := baseDirectory.value / s"target/${name.value}-${version.value}.jar",
       testFrameworks += new TestFramework("zio.test.sbt.ZTestFramework"),
       jacocoReportSettings := jacocoSettings(scalaVersion.value, "atum-server"),
-      serverMergeStrategy
+      serverMergeStrategy,
+      commands += Command.command("testIT") { state =>
```

Comment:
I don't fully understand this code yet. But having "aliases" defined in different places i s not the nicest - would try to voidi that if possible. What is the goal of having this here?

Reply:
This solution is parallel to aliases one. This one is used only for Zio tests in the server module.
We have two test control solutions using two test frameworks in the project!!!
I could not find how to control the exclusion of Integration/DB tests by one command.

Reply:
After refactoring, only the "aliases" solution was kept, and the QA Type system in the test file suffix naming.